### PR TITLE
feat: how to use sern blog post

### PR DESCRIPTION
**Pull request by SrIzan10**
yeahhhhhhhhhh im bored help

### CODE REVIEW

File: blog/2022-09-30-mdx-blog-post.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## FAQ
+
+**Q**: How do I publish a slash command?  
+**A**: Install the publish extension. Little video:  
+<video width="400" controls>
+  <source src="https://srizan.s-ul.eu/uHzPhfcS" type="video/mp4" />
+</video>
+
+**Q**: Any snippet VSCode extension?  
+**A**: Yeah, just search `sern Snippets` made by a verified publisher called Sr Izan (haha yeah me funny!1!)
```
Comment (jacoobes) [inline blog/2022-09-30-mdx-blog-post.md]: typo: !1! -> !!!

File: blog/2022-09-30-mdx-blog-post.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+sern init
+```
+The CLI is written in Typescript and open-sourced on [Github](https://github.com/sern-handler/cli). (thanks evo!)
+
+### Step 2: Have some way to store secrets.
+
+Normally you'd need a way to store secrets, and the best way to do that is by installing another package: `dotenv`
+
+just `npm i dotenv` in the project folder and add `require('dotenv').config()` to your import section. Then, when you login, `process.env.TOKEN` (or however you have it named on your `.env` file) should do the trick.
```
Comment (jacoobes) [inline blog/2022-09-30-mdx-blog-post.md:34]: Add an admonition saying require('dotenv') doesnt work for esm

Comment (jacoobes) [inline blog/2022-09-30-mdx-blog-post.md:34]: don't give exact steps as users may not use the same methods. Give a  general overview. For example, providing the steps to this is a little too long

File: blog/2022-09-30-mdx-blog-post.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+slug: getting-started
+title: How to get started with sern!
+authors: [Sr Izan]
+tags: [guides]
+---
+
+## Hello everyone!
+
+I'm Sr Izan, your fellow sern-handler user and contributor.
```
Comment (jacoobes) [inline blog/2022-09-30-mdx-blog-post.md]: remove sern-handler

File: blog/2022-09-30-mdx-blog-post.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Today I'm going to show you how to get started with sern and all its cool features.
+
+### Step 1: Install the CLI.
+
+Just run:
+```
+npm i -g @sern/cli
+```
+then just use
```
Comment (jacoobes) [inline blog/2022-09-30-mdx-blog-post.md]: then run, not then just use. also add admonition for sern init -y

**Review by jacoobes — CHANGES_REQUESTED** [review]

File: blog/2022-09-30-mdx-blog-post.md (hunk 1 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md]: yea its intentional 😅

File: blog/2022-09-30-mdx-blog-post.md (hunk 2 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md:34]: kay

File: blog/2022-09-30-mdx-blog-post.md (hunk 4 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md:20]: if you want I can make it a tip instead

File: blog/2022-09-30-mdx-blog-post.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+slug: getting-started
```
Comment (Murtatrxx) [inline blog/2022-09-30-mdx-blog-post.md:2]: Does this needs to be kebab case?

File: blog/authors.yml [hunk 6]
```diff
@@ -2,4 +2,9 @@ jacoobes:
...
   title: Head Dev
   url: https://github.com/jacoobes
-  image_url: https://github.com/jacoobes.png
\ No newline at end of file
+  image_url: https://github.com/jacoobes.png
+Sr Izan:
+  name: Sr Izan
+  title: Contributor
+  url: https://github.com/SrIzan10
+  image_url: https://github.com/SrIzan10.png
```
Comment (Murtatrxx) [inline blog/authors.yml]: If you can, please add me there, I might post a text in future. If you don't want to do, dw I'll when I want to post something

**Review by Murtatrxx — APPROVED** [review]

File: blog/2022-09-30-mdx-blog-post.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Just run:
+```
+npm i -g @sern/cli
+```
+then just use
+```
+sern init
+```
+The CLI is written in Typescript and open-sourced on [Github](https://github.com/sern-handler/cli). (thanks evo!)
```
Comment (Murtatrxx) [inline blog/2022-09-30-mdx-blog-post.md]: Ping evo's gh

File: blog/2022-09-30-mdx-blog-post.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+**A**: Install the publish extension. Little video:  
+<video width="400" controls>
+  <source src="https://srizan.s-ul.eu/uHzPhfcS" type="video/mp4" />
+</video>
+
+**Q**: Any snippet VSCode extension?  
+**A**: Yeah, just search `sern Snippets` made by a verified publisher called Sr Izan (haha yeah me funny!1!)  
+
+**Q**: HEEEELLLPPPP!!!!
+**A**: Hey, don't panic! We're here to help so, join the [Discord](https://sern-handler.js.org/discord). We're trying to get to 100 members!
```
Comment (Murtatrxx) [inline blog/2022-09-30-mdx-blog-post.md]: Idk why but answer doesn't prints in a new line, they both print in the same line

**Review by Murtatrxx — CHANGES_REQUESTED** [review]

**Comment by Murtatrxx** [comment]
And overall, please be more serious

**Comment by jacoobes** [comment]
I kinda like the tone though, it gets to the point, blogs don't have to be serious

**Comment by SrIzan10** [comment]
exactly, the blog post wasn’t going to be serious at all!

**Comment by Murtatrxx** [comment]
🫤

**Comment by Murtatrxx** [comment]
Ok, then

**Comment by Murtatrxx** [comment]
Please make the changes I requested

File: blog/2022-09-30-mdx-blog-post.md (hunk 7 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md]: is this a thing in docusaurus?

File: blog/2022-09-30-mdx-blog-post.md (hunk 8 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md]: This can be fixed by adding two spaces after the end of the lines, i'll see if thst also happens on my end

**Comment by SrIzan10** [comment]
I've been busy lately, so I'll finish all of this when I have time

File: blog/2022-09-30-mdx-blog-post.md (hunk 5 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md:2]: I'll leave it as kebab-case for now. @jacoobes reply to this if you want anther case type

File: blog/authors.yml (hunk 6 above)
Comment (SrIzan10) [reply blog/authors.yml]: Yeah cool

**Comment by SrIzan10** [comment]
also please merge sern-handler/templates#17 when this also gets merged

File: blog/2022-09-30-mdx-blog-post.md (hunk 2 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md:34]: done

File: blog/2022-09-30-mdx-blog-post.md (hunk 3 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md]: cool

File: blog/2022-09-30-mdx-blog-post.md (hunk 2 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md:34]: I'll make an admonition for ESM users and that's it, it's not very long I guess

File: blog/2022-09-30-mdx-blog-post.md (hunk 4 above)
Comment (SrIzan10) [reply blog/2022-09-30-mdx-blog-post.md]: done

**Comment by SrIzan10** [comment]
strange, can't re-request review for you two 🤔

**Review by Murtatrxx — APPROVED** [review]

**Comment by Murtatrxx** [comment]
It looks ok, will merge it soon. Thanks for your work.

**Comment by SrIzan10** [comment]
thanks! 🎉

**Comment by SrIzan10** [comment]
yeah i’ll add it 👍

**Pull request merged**